docs/src/tutorials/eeg_animation.jl

```diff
@@ -0,0 +1,284 @@
+using Javis
```

Comment:
Is this file linked somewhere?

Reply:
Not yet - I was going to put it into the JavisTemplates repo later and link that back here.